test/Search_UnitTestHelpers.hpp

```diff
@@ -103,6 +103,19 @@ auto query(ExecutionSpace const &exec_space, Tree const &tree,
       Kokkos::create_mirror_view_and_copy(Kokkos::HostSpace{}, values));
 }
 
+template <typename ExecutionSpace, typename MemorySpace, typename Queries>
+auto query(ExecutionSpace const &exec_space,
+           ArborX::BVH<MemorySpace> const &tree, Queries const &queries)
+{
+  using memory_space = MemorySpace;
+  Kokkos::View<int *, memory_space> values("Testing::values", 0);
+  Kokkos::View<int *, memory_space> offsets("Testing::offsets", 0);
+  ArborX::query(tree, exec_space, queries, values, offsets);
+  return make_compressed_storage(
+      Kokkos::create_mirror_view_and_copy(Kokkos::HostSpace{}, offsets),
+      Kokkos::create_mirror_view_and_copy(Kokkos::HostSpace{}, values));
+}
+
```

Comment:
```suggestion
```
You don't need that overload

Reply:
It is used when comparing with boost rtree, and only for rtree. I'm OK with keeping them, but can change if you insist.

Reply:
Does the function declared above it not work?

Reply:
Things do not compile with this function removed.

Reply:
Correction, things compile, but with a warning. Essentially, `ARBORX_TEST_QUERY_TREE` then hits ```
template <typename ExecutionSpace, typename Tree, typename Queries>                                                                     
auto query(ExecutionSpace const &exec_space, Tree const &tree,                                                                          
           Queries const &queries)                                                                                                      
```
function that has
```
tree.query(exec_space, queries, values, offsets);
```
inside. But as this will be called with both on-node and distributed tree, it cannot be changed to `ArborX::query` easily.

Reply:
Comment that this is a temporary workaround until we (hopefully) reconcile the interface of the distributed tree with the on-node tree.

Reply:
Done.